packages/visualizations/src/components/Table/Cell/Format/GeoFormat.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+    import { tick } from 'svelte';
+    import { WebGlMap } from 'components/Map';
+    import tippy from 'components/utils/tippy';
+
+    import type { WebGlMapOptions, WebGlMapData } from 'components/Map';
+
+    export let rawValue: unknown;
+    export let display = (v: unknown) => v;
+    export let mapOptions: WebGlMapOptions;
+    export let sources: (v: unknown) => WebGlMapData['sources'] = () => ({});
+    export let layers: (v: unknown) => WebGlMapData['layers'] = () => [];
+
+    let tooltipContent;
+    let showMap;
+
+    $: data = {
+        sources: sources(rawValue),
+        layers: layers(rawValue),
+    };
+</script>
+
+<div
+    role="button"
+    tabindex="0"
+    use:tippy={{
+        content: tooltipContent,
+        theme: 'ods-visualization-table',
+        delay: [500, 0],
+        duration: [275, 0],
+        maxWidth: 'none',
+        onTrigger: (instance) => {
+            showMap = true;
+        },
+        onUntrigger: (instance) => {
+            showMap = false;
+        },
```

Comment:
Only way that I found to render properly the tooltip.
Without it, the tooltip is not visible when events (focus, mousenter) are triggered